test/test_systems_generators_dockerfile.py

```diff
@@ -266,11 +267,11 @@ def test_buildDockerfile_custom(self, mock_getcwd, mock_expanduser):
 FROM whatever:uwant
 MAINTAINER Mega Man <megaman@cars.com>
 WORKDIR /app
-RUN ["pip", "install", "pyyaml"]
-RUN ["pip", "install", "artifactory"]
-RUN ["pip", "install", "argparse"]
-RUN ["pip", "install", "coverage"]
-RUN ["pip", "install", "pytest"]
+RUN ["pip3", "install", "pyyaml"]
+RUN ["pip3", "install", "artifactory"]
+RUN ["pip3", "install", "argparse"]
+RUN ["pip3", "install", "coverage"]
+RUN ["pip3", "install", "pytest"]
```

Comment:
Still need to support `pip` installs for the Python base images.

Reply:
updated